### ISSUE TITLE
SPM coregistration generates a very large file 

### ISSUE THREAD

**Issue by smortaheb** [nipy/nipype#3229]
### Summary
Hello,
When I try to apply SPM coregisteration (native fMRI to the native T1) in nipype, (jobtype = ‘estwrite’) the output coregistered file will have a very large volume. For example, my functional file originally is ~50 MB. But after coregisteration it becomes ~4GB. How can I solve this problem? (When I perform coregistration separately inside the SPM12 GUI, there is no such a problem)

### Script/Workflow details

ra = Realign() 
ra.inputs.in_files = 'func.nii' 
ra.inputs.register_to_mean = True 
ra.run()

coreg = Coregister() 
coreg.inputs.source = 'meanfunc.nii' 
coreg.inputs.target = 'struct.nii' 
coreg.inputs.apply_to_func = 'rfunc.nii' 
coreg.run()

### Platform details:
```
{'commit_hash': '%h',
 'commit_source': 'archive substitution',
 'networkx_version': '2.3',
 'nibabel_version': '3.0.0',
 'nipype_version': '1.3.1',
 'numpy_version': '1.17.2',
 'pkg_path': '/home/sepehr/anaconda3/lib/python3.7/site-packages/nipype',
 'scipy_version': '1.3.1',
 'sys_executable': '/home/sepehr/anaconda3/bin/python',
 'sys_platform': 'linux',
 'sys_version': '3.7.4 (default, Aug 13 2019, 20:35:49) \n[GCC 7.3.0]',
 'traits_version': '5.2.0'}

```

### Execution environment

Choose one
- My python environment outside container

**Comment by satra** [comment]
@gllmflndn - any thoughts here? did the setting on SPM change?

@smortaheb - when you run the above command a pyscript.m file would have been produced. could you please upload that file somewhere where we can take a look at, or paste it here. you can use a structure like this to minimize the display

```
<details> <summary> coreg m file </summary>

enter content here
</details>
```

**Comment by smortaheb** [comment]
@satra 
I think I figured out the problem. My functional data has a dimension of 64*64*32*200 and my structural data has a dimension of 208*256*192 and so by setting jobtype='estwrite' it performs resampling which results in producing a functional file with dimensions of 206*256*192*200 which has 80 times more voxels than the original file and so a volume of 80 times greater than the original one. (50 MB * 80 ~= 4 GB). To solve the problem I just set the jobtype to 'estimate' to prevent resampling in this step which is unnecessary.
